component/storer.go

```diff
@@ -184,7 +184,7 @@ func makeStorer(
 				}
 				err := addresses.Update(wasting)
 				if err != nil {
-					return errors.Wrap(err, "failed to insert wasting")
+					return errors.Wrap(err, "failed to insert vasting")
```

Comment:
vasting ?

Reply:
vesting

Reply:
Yep. I think yes. I get from this https://cvj.ch/en/glossary/vesting/

Reply:
fix it on vesting